### PR TITLE
MB Homepage Revamp

### PR DESCRIPTION
**Pull request by akshaaatt**
This implements a modern look to the MusicBrainz Web Homepage.

Discussion available at: https://community.metabrainz.org/t/musicbrainz-web-homepage-2021/554362

### CODE REVIEW

File: root/main/components/Home/About/About.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import './About';
```
[the remark this answers is not in the document]
Comment (akshaaatt) [reply root/main/components/Home/About/About.js]: Life saver. Thanks!

File: root/main/components/Home/About/About.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    <section className={'about ' + theme} id="about">
+      <div className="container">
+
+        <div className="row">
+          <div className="col-lg-6 order-1 order-lg-2" data-aos="zoom-in" data-aos-delay="150">
+            <img alt="" className="img-fluid" src="assets/img/music.jpg" />
+          </div>
+          <div className="col-lg-6 pt-4 pt-lg-0 order-2 order-lg-1 content" data-aos="fade-right">
+            <h3 className="navbar-brand text-brand">
+              About
```
[the remark this answers is not in the document]
Comment (akshaaatt) [reply root/main/components/Home/About/About.js:34]: Useful! All done.

File: package.json [hunk 3]
```diff
@@ -49,20 +53,27 @@
     "pg": "8.3.3",
     "pg-cursor": "2.3.3",
     "po2json": "0.4.1",
+    "prop-types": "^15.7.2",
```
[the remark this answers is not in the document]
Comment (akshaaatt) [reply package.json:54]: Removed

**Comment by mwiencek** [comment]
Hi @akshaaatt, we took a look at this PR as a team, but it's way too big and unwieldy as it is, so we'd like to have a meeting with you where we spend a couple hours picking a section of it and integrating that into the current homepage.  
 There are some code issues here that require mentorship, so it'll be much more manageable if we can break this into smaller chunks and work together on smaller PRs.  But we'll have to determine the best way to do that without making the homepage schizophrenic.  I'll ping you on IRC about setting up a meeting.  :)

**Comment by akshaaatt** [comment]
Smaller PRs are being worked on to update the Homepage :)

**Pull request closed without merging**